package.json

```diff
@@ -87,7 +87,14 @@
 		"typescript-4-5-4": "npm:typescript@4.5.4",
 		"typescript-4-6-4": "npm:typescript@4.6.4",
 		"typescript-4-7-2": "npm:typescript@4.7.2",
-		"typescript-4-8-2": "npm:typescript@4.8.2"
+		"typescript-4-8-2": "npm:typescript@4.8.2",
+		"typescript-4-9-4": "npm:typescript@4.9.4",
+		"typescript-4-9-5": "npm:typescript@4.9.5",
+		"typescript-5-0-3": "npm:typescript@5.0.3",
+		"typescript-5-0-4": "npm:typescript@5.0.4",
+		"typescript-5-1-3": "npm:typescript@5.1.3",
+		"typescript-5-1-5": "npm:typescript@5.1.5",
+		"typescript-5-1-6": "npm:typescript@5.1.6"
```

Comment:
Awesome, but I think we should keep it at one TS version per minor release, so I'd suggest extending it with simply:

```
"typescript-4-9-4": "npm:typescript@4.9.4",
"typescript-5-0-4": "npm:typescript@5.0.4",
"typescript-5-1-6": "npm:typescript@5.1.6"
```